src/Microsoft.AspNetCore.Mvc.TagHelpers/ValidationMessageTagHelper.cs

```diff
@@ -71,11 +71,13 @@ public ValidationMessageTagHelper(IHtmlGenerator generator)
                     };
                 }
 
+                var childContent = await output.GetChildContentAsync();
```

Comment:
Set `childContent` to `null` if `output.IsContentModified`; the `GetChildContentAsync()` call is expensive.

Reply:
Yeah sure.